src/Microsoft.PowerApps.TestEngine/Modules/TestEngineExtensionChecker.cs

```diff
@@ -288,6 +317,45 @@ public bool VerifyContainsValidNamespacePowerFxFunctions(TestSettingExtensions s
 
                 foreach (TypeDefinition type in module.GetAllTypes())
                 {
+                    // Provider checks are based on Namespaces string[] property
+                    if (
+                        type.Interfaces.Any(i => i.InterfaceType.FullName == "Microsoft.PowerApps.TestEngine.Providers.ITestWebProvider")
+                        ||
+                        type.Interfaces.Any(i => i.InterfaceType.FullName == "Microsoft.PowerApps.TestEngine.Users.IUserManager")
+                        )
+                    {
+                        if (CheckPropertyArrayContainsValue(type, "Namespaces", out var values))
```

Comment:
If "Namespace" is used multiple times, let's define a const

Reply:
Updated